src/rendering/screenspacerenderable.cpp

```diff
@@ -51,17 +51,17 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo EnabledInfo = {
         "Enabled",
         "Enabled",
-        "This setting determines whether this sceen space plane will be visible or not.",
+        "Determines whether this sceen space object will be rendered or not.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
     constexpr openspace::properties::Property::PropertyInfo RenderDuringBlackoutInfo = {
         "RenderDuringBlackout",
         "Render during Blackout",
-        "If this value is 'true', this screenspace renderable is going to ignore the "
-        "global blackout factor from the Render Engine and will always render at full "
-        "opacity. If it is 'false', it will adhere to the factor and fade out like the "
```

Comment:
Technically, I guess full opacity means fully transparent, but I don't know if that is how it is actually used (in css for example)

Reply:
No, the opposite.  Full opacity means not transparent at all